File: Assignment_No19/File1.c
#include<stdio.h>
/*
A B C D
A B C D
A B C D
A B C D
*/
void Pattern(int iRows, int iCol)
{
    int i=0, j=0;
    char ch = '\0';

    for(i=1 ; i<=iRows; i++)
    {
        for(j=1, ch='A'; j<=iCol; j++, ch++)
        {
            printf("%c\t", ch);
        }
        printf("\n");
    }
}
int main()
{
    int iValue1=0 , iValue2=0;

    printf("Enter the number of rows and columns\n");
    scanf("%d %d",&iValue1,&iValue2);

    Pattern(iValue1,iValue2);

    return 0;
}

File: Assignment_No47/File1.java
import java.util.*;
import LB.Matrix;

class File1
{
    public static void main(String[] args)
    {
        int iRow = 0, iCol = 0, iRet = 0;
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the number of rows");
        iRow = sobj.nextInt();

        System.out.println("Enter the number of columns");
        iCol = sobj.nextInt();

        MyMatrix mobj = new MyMatrix(iRow,iCol);

        mobj.Accept();
        mobj.Display();

        mobj.Transpose();
    }
}

class MyMatrix extends Matrix
{
    public MyMatrix(int A, int B)
    {
        super(A,B);
    }

    public void Transpose()
    {
        int Brr[][] = new int[3][3];
        int iTemp = 0;
        int i = 0, j = 0;
        for(i=0; i<Arr.length ; i++)
        {
            for(j=0; j<Arr[i].length; j++)
            {
                Brr[i][j] = Arr[j][i];
            }
        }

        System.out.println("Transpose of a matrix is :\n");
        for(i = 0; i < Arr.length; i++)
        {
            for(j = 0; j < Arr[i].length; j++)
            {
                System.out.print(Brr[i][j]+"\t");
            }
            System.out.println();
        }

    }
}